src/encoded/commands/example_command.py

```diff
@@ -0,0 +1,34 @@
+import argparse
```

Comment:
Unclear what this is for

Reply:
Probably I was going to make the example command take arguments. This command can go away, or can be used in unit testing. I was using it for interactive testing to make sure setup was done right. Its value to me is gone away.